code/modules/trade/datums/trade_stations_presets/serbianship.dm

```diff
@@ -0,0 +1,26 @@
+/datum/trade_station/serbfrigate
+	icon_states = "serb_frigate"
+	spawn_cost = 2
+	spawn_probability = 10
+	markup = 3
+	name_pool = list(
+		"STF 'Kovac'" = "Serbia Trade Frigate 'Kovac', *generic description for generic serbians alpha ape males that ain't killing you instantly*",
+		"SV 'Zoric'" = "Serbia Vessel 'Zoric' Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
```

Comment:
Why are they named after the guns?

Reply:
For the same reason the description is like that. I got not better idea for now.

~~Actual headcanon: Serbians think ships are big guns and themselves as bullets, so guns shoot serbians into ships/stations~~